README.md

```diff
@@ -8,6 +8,10 @@ For more informations, check our [Runbook](https://dsdmoj.atlassian.net/wiki/spa
 
 ---
 
+## Testing
+
+`docker compose up localstack-court-case-service postgres`
```

Comment:
I see. Remove test containers localstack and instead run localstack in a docker container externally. 🥇 awesome thank you